webnode/config/webpack.config.dev.js

```diff
@@ -222,46 +126,5 @@ module.exports = {
         "eth-address": "0xD1833A50f411432aD38E8374df8Cfff79e743788"
       }
     }),
-    new webpack.EnvironmentPlugin({
-      NODE_ENV: "development",
-      DEBUG: true
-    }),
```

Comment:
we need the `webpack.EnvironmentPlugin` stuff

Reply:
ill add this back